test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Date.Tests.ps1

```diff
@@ -37,6 +37,10 @@ Describe "Get-Date DRT Unit Tests" -Tags "CI" {
         Get-date -Date 1/1/0030 -uformat %S%T%u%U%V%w%W%x%X%y%Y%% | Should be "0000:00:002012001/01/3000:00:00300030%"
     }
 
+    It "using -uformat with an empty format string produces a descriptive error" {
+        { Get-date -Date 1/1/1970 -uformat '' -ErrorAction Stop } | ShouldBeErrorId "InvalidUFormatString,Microsoft.PowerShell.Commands.GetDateCommand"
```

Comment:
Perhaps have two `-TestCases` where one is an empty string and the other is just whitespace